src/components/backend-ai-resource-monitor.ts

```diff
@@ -704,6 +704,10 @@ export default class BackendAiResourceMonitor extends BackendAIPage {
     }
   }
 
+  _calcConcurrencyMax() {
+    return this.concurrency_max === 1000000 ? `∞` : this.concurrency_max;
```

Comment:
If it is added to remove type miscast, it should be 

```
return this.concurrency_max === 1000000 ? `∞` : parseInt(this.concurrency_max);
```

Reply:
Got it! I'll keep this in mind. Thank you!